src/js/components/Scope/DatasetScopePicker.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { List, Avatar, Space, Typography } from 'antd';
+import { useTranslationCustom, useTranslationDefault } from '@/hooks';
+import { Link, useLocation } from 'react-router-dom';
+import { FaDatabase } from 'react-icons/fa';
+import { getCurrentPage } from '@/utils/router';
+import type { Project } from '@/types/metadata';
+
+type DatasetScopePickerProps = {
+  parentProject: Project;
+  isSingleProject?: boolean;
+};
+export const DatasetScopePicker = ({ parentProject, isSingleProject }: DatasetScopePickerProps) => {
+  const td = useTranslationDefault();
+  const t = useTranslationCustom();
+  const location = useLocation();
+  const baseURL = '/' + location.pathname.split('/')[1];
+  const page = getCurrentPage();
+  return (
+    <Space direction="vertical" style={{ display: 'flex' }}>
+      <Space align="baseline" size="large">
+        <Typography.Title level={4} className="no-margin-top">
+          {td('Project')}: {t(parentProject.title)}
+        </Typography.Title>
+        {isSingleProject ? (
+          // give clear scope option in single project mode
+          <Link to={baseURL}>
+            <Typography.Link>{td('Clear')}</Typography.Link>
+          </Link>
+        ) : (
+          <Link to={`${baseURL}/p/${parentProject.identifier}/${page}`} key="3">
+            <Typography.Link>{td('Select')}</Typography.Link>
+          </Link>
+        )}
+      </Space>
+      <Typography.Text>{t(parentProject.description)}</Typography.Text>
+      <Typography.Title level={5} className="no-margin-top">
+        {td('Datasets')}
+      </Typography.Title>
+      <List
+        dataSource={parentProject.datasets}
+        bordered
+        renderItem={(item) => (
+          <Link to={`${baseURL}/p/${parentProject.identifier}/d/${item.identifier}/${page}`}>
+            <List.Item className="select-dataset-hover" key={item.identifier}>
```

Comment:
this is out-of-scope for this PR, perhaps, but there should be a class for whichever dataset is currently selected giving an effect similar to hover (via CSS) visually indicating the selected dataset.

Reply:
I went with changing the dataset icon color based on the selected dataset scope.
![image](https://github.com/user-attachments/assets/5e379484-cc60-4a57-8789-2844d6619ef9)